### PR TITLE
MNT: Update feedstock

### PR DESCRIPTION
**Pull request by kmuehlbauer**
+ rerender smithy 1.7.0
+ update pinning
+ long prefix build

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
@ocefpaf Could you have a final look here?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,7 +30,6 @@ test:
     - test -e $PREFIX/lib/libradolan.dylib  # [osx]
     - radolan2netcdf --version
     - radolan2netcdf --endianess
-    - conda inspect linkages -n _test libradolan  # [not win]
```
Comment (ocefpaf) [inline recipe/meta.yaml:33]: If you are using `conda-build 2` you can add here:

```yaml
    - conda inspect linkages -p $PREFIX libradolan  # [not win]
    - conda inspect objects -p $PREFIX libradolan  # [osx]
```

File: ci_support/run_docker_build.sh [hunk 2]
```diff
@@ -41,7 +41,14 @@ conda clean --lock
 conda install --yes --quiet conda-forge-build-setup
 source run_conda_forge_build_setup
 
+# install conda-build 2.x to build a long prefix
+conda install --yes --quiet conda-build=2
```
Comment (ocefpaf) [inline ci_support/run_docker_build.sh:45]: :+1:

**Comment by ocefpaf** [comment]
> @ocefpaf Could you have a final look here?

LGTM. I made one minor comment that is up to you if you want to add it or no.

**Pull request merged**